packages/client/tests/unit.utils.test.ts

```diff
@@ -1,3 +1,4 @@
+import { it, expect, describe } from 'vitest';
```

Comment:
These same imports are on line 5. As a more general observation, there seems to be a bit of noise in this PR related to vitest setup. Is there some carry-over from a weird merge from #12530 in this PR? Seems like we could use some cleanup.